internal/encoding/graphite/config.go

```diff
@@ -0,0 +1,161 @@
+package graphite
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/influxdata/influxdb/models"
+)
+
+const (
+	// DefaultSeparator is the default join character to use when joining multiple
+	// measurment parts in a template.
+	DefaultSeparator = "."
+)
+
+// Config represents the configuration for Graphite endpoints.
+type Config struct {
+	Separator string
+	Tags      []string
```

Comment:
Please remove the `Tags` capability from the graphite parser. Telegraf already has the capability to specify per-plugin tags: https://github.com/influxdata/telegraf/blob/master/CONFIGURATION.md#input-config-tags